src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/help-checkoutCredentialsId.html

```diff
@@ -1,3 +1,15 @@
 <div>
-    <p>Credentials used to check out sources during a build.</p>
+    <p>
+        Credentials used to check out sources during a build.
+    </p>
+    <p>
+        The default choice of "Same as scan credentials" is sufficient so long as you don't
+        need Jenkins to push some changes back into these repositories.
+    </p>
+    <p>
+        Otherwise you should create an <strong>SSH username with private key</strong> credential and
```

Comment:
So this is actually a good conversation, because this is the kind of thing docs should clarify.

I know you _can_ specify HTTP credential (as in personal access token), but my question is, why would you. I think that case is covered by the paragraph above about "same as scan credentials". I cannot think of any situation where you want to use a separate HTTP credential for a checkout credential.

My point that I wanted to emphasize in the doc is that SSH credential as the checkout credential has a distinct use here, which is about making commits and pushing them. That's worth calling out for.